packages/react-native-reanimated/src/layoutReanimation/web/createAnimation.ts

```diff
@@ -97,41 +137,26 @@ export function createAnimationWithInitialValues(
   const animationStyle = structuredClone(AnimationsData[animationName].style);
   const firstAnimationStep = animationStyle['0'];
 
-  const { transform, ...rest } = initialValues;
-  const transformWithPx = addPxToTransform(transform as TransformType);
+  const { transform, originX, originY, ...rest } = initialValues;
+
+  const transformStyle = buildTransformMap(firstAnimationStep.transform);
```

Comment:
```suggestion
  const transformStyle = new Map<string, TransformValue>();
  assignTransformRules(transformStyle, firstAnimationStep.transform);
```

From [this](https://github.com/software-mansion/react-native-reanimated/pull/8733/changes#r2621384647) suggestion comment